build/gke-test-cluster/cluster.tf

```diff
@@ -0,0 +1,153 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Should this be in "build" - given that this will likely be our 'here's our opinionated way to install a cluster + Agones' - should this be in a more central place? What do you all think?

Reply:
Yes, I can move this up to a "build" folder. Thinking on how to make this method of creating cluster universal (cloud platform agnostic), but this definitely would be separate change.